warden/warden_daemon/src/main.rs

```diff
@@ -0,0 +1,12 @@
+use anyhow::Error;
+use clap::Parser;
+use warden_daemon::{app::App, cli::Cli};
+
+#[tokio::main(flavor = "multi_thread")]
+async fn main() -> anyhow::Result<(), Error> {
+    env_logger::init();
+    let cli = Cli::parse();
+
+    let app = App::new(cli).await?;
+    app.run().await?.await?
```

Comment:
Minor: This looks funny. Can we assign the return value from run so that we know what is being awaited (some thread handle?)?

Reply:
Done